controllers/userController.js

```diff
@@ -183,11 +184,22 @@ const verifyResetPasswordToken = async (req, res) => {
     } else {
       res.status(200).send({ message: "user password reset token expired" })
     }
-
-
   } catch (err) {
     res.status(500).send(err)
   }
-
 }
-module.exports = { userRegister, login, getUserData, deleteUserData, limitUsersData, userAddress, deleteUserAddresses, forgotPassword, verifyResetPasswordToken }
+const addUserProfileImage = async (req, res) => {
+  const username = req.params.username;
```

Comment:
why ask the user to give you the user name? You can get the user's id  from the token, use that @ashok-choudhary-0